companion/src/modeledit/logicalswitches.h

```diff
@@ -37,12 +38,16 @@ class LogicalSwitchesPanel : public ModelPanel
     virtual void update();
 
   private slots:
-    void edited();
+    void setDataModels();
+    void functionChanged();
     void v1Edited(int value);
     void v2Edited(int value);
     void andEdited(int value);
     void durationEdited(double duration);
     void delayEdited(double delay);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS1144)